recipes/mgis/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "mgis" %}
+{% set version = "2.1" %}
+{% set mfront_ver = "4.1.0" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  - fn: '{{ name }}_{{ version }}.tar.gz'
+    url: https://github.com/thelfer/MFrontGenericInterfaceSupport/archive/refs/tags/MFrontGenericInterfaceSupport-{{ version }}.tar.gz
+    sha256: f5b556aab130da0c423f395fe4c35d6bf509dd8fc958242f2e37ea788464aea9
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  skip: true  # [py2k or osx or win or (python_impl == 'pypy')]
+
+requirements:
+  build:
+    - "{{ compiler('c') }}"
+    - "{{ compiler('cxx') }}"
+    - "{{ compiler('fortran') }}"
+    - cmake >=3.26.4
+    - autoconf
+    - python
+    - make
+
+  host:
+    - python
+    - boost 1.78
+    - numpy 1.23
+    - mfront {{ mfront_ver }}
```

Comment:
Why is this version templated?

Reply:
Yeah, this isn't really necessary. 

My reason for doing so originally is that mgis is an adapter library for mfront and at the time I thought it would be helpful to set the mgis version together with the mfront version at the top

```
{% set version = "2.1" %}
{% set mfront_ver = "4.1.0" %}
```

But I have no strong preferences on this, so I am fine with changing it :)

Reply:
I changed this to a regular version pinning. Hope it's okay I checked this as resolved